hooks/useIpfsSubmit.ts

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react'
+import axios, { AxiosError } from 'axios'
+import { useToast } from '@/hooks/useToast'
+import { useGetPoint } from '@/hooks/quest/useGetPoint'
+
+export interface TokenAttribute {
+  display_type: string | undefined
+  trait_type: string
+  value: number | string
+}
+export interface KamonToken {
+  name: string
+  description: string
+  image: string
+  attributes: TokenAttribute[]
+}
+
+export const useIpfsSubmit = () => {  
```

Comment:
`useUpdateImageForPoint`

Reply:
How about `useUpdateTokenMetadata`?

- This doesn't just update the image but the whole metadata
- This hook can be used later, when we update NFTs for reasons other than the point total